attributes/default.rb

```diff
@@ -305,7 +305,7 @@
 default['cluster']['efs_utils']['sha256'] = '69d0d8effca3b58ccaf4b814960ec1d16263807e508b908975c2627988c7eb6c'
 default['cluster']['efs_utils']['tarball_path'] = "#{node['cluster']['sources_dir']}/efs-utils-#{node['cluster']['efs_utils']['version']}.tar.gz"
 default['cluster']['stunnel']['version'] = '5.67'
-default['cluster']['stunnel']['url'] = "https://www.stunnel.org/downloads/stunnel-#{node['cluster']['stunnel']['version']}.tar.gz"
+default['cluster']['stunnel']['url'] = "#{node['cluster']['artifacts_s3_url']}/stunnel/stunnel-5.67.tar.gz"
```

Comment:
Please reuse `#{node['cluster']['stunnel']['version']}` for the version

Reply:
Do you mean on line 307 as `default['cluster']['stunnel']['version'] = #{node['cluster']['stunnel']['version']}`

Reply:
Or on line 308 as `default['cluster']['stunnel']['url'] = "#{node['cluster']['artifacts_s3_url']}/stunnel/stunnel-#{node['cluster']['stunnel']['version']}.tar.gz `

Reply:
Change line 308

Reply:
Ok, I changed that here [reuse stunnel version variable](https://github.com/aws/aws-parallelcluster-cookbook/pull/1623/commits/1ed48da4b685ecfdffbb31c1e53ea2eb81839851)